ClearKeep/Scenes/InCall/CallVideoActionView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CallActionBottomView.swift
+//  ClearKeep
+//
+//  Created by Luong Minh Hiep on 18/05/2021.
+//
+
+import SwiftUI
+
+struct CallVideoActionView: View {
+    @ObservedObject var viewModel: CallViewModel
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/InCall/CallVideoActionView.swift#L11"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>CallVideoActionView.swift:11</code></td>
    </tr>
  </tbody>
</table>